shell/common/vsync_waiter_fallback.cc

```diff
@@ -35,13 +35,20 @@ VsyncWaiterFallback::~VsyncWaiterFallback() = default;
 
 // |VsyncWaiter|
 void VsyncWaiterFallback::AwaitVSync() {
-  TRACE_EVENT0("flutter", "VSYNC");
-
   constexpr fml::TimeDelta kSingleFrameInterval =
       fml::TimeDelta::FromSecondsF(1.0 / 60.0);
   auto frame_start_time =
       SnapToNextTick(fml::TimePoint::Now(), phase_, kSingleFrameInterval);
   auto frame_target_time = frame_start_time + kSingleFrameInterval;
+
+  const std::string start_time_str =
+      std::to_string(frame_start_time.ToEpochDelta().ToMicroseconds());
+  const std::string frame_target_time_str =
+      std::to_string(frame_target_time.ToEpochDelta().ToMicroseconds());
+  TRACE_EVENT2("flutter", "VsyncAlike", "frame_start_time",
```

Comment:
It might be worth making a new `TRACE_EVENT2_INT` macro that does this for you. It's used at multiple places.

Reply:
Sorry but I cannot find it :/ https://github.com/search?q=repo%3Aflutter%2Fengine+TRACE_EVENT2_INT&type=code https://github.com/search?q=repo%3Aflutter%2Fengine+TRACE_EVENT_INT2&type=code https://github.com/search?q=repo%3Aflutter%2Fengine+TRACE_EVENT_INT&type=code

Reply:
I meant that it would be useful to create one such macro 😊

Reply:
Oh yes, sure!